src/api/resourceApi.ts

```diff
@@ -80,3 +80,15 @@ export const getResourceContents = (identifier: string): Promise<AxiosResponse<C
     method: 'GET',
   });
 };
+
+export const updateContentTitle = async (resourceIdentifier: string, contentIdentifier: string, value: string) => {
+  const encodedValue = encodeURIComponent(value);
```

Comment:
hvis denne skal sendes i body tror jeg ikke den burde bruke encodeURIComponent..? det er tross alt ikke en del av URIen

Reply:
tror faktisk det,desverre.  "A value of "application/x-www-form-urlencoded" means that your POST body will need to be URL encoded just like a GET parameter string. "

Reply:
ooh, det blir kanskje annerledes når man ikke sender som JSON ja...